package.json

```diff
@@ -32,12 +32,12 @@
   "scripts": {
     "lint": "sewing-kit lint",
     "format": "sewing-kit format",
-    "ts": "tsc --noEmit",
+    "type-check": "sewing-kit type-check",
```

Comment:
What does sewing-kit run under the hood?

Reply:
It still runs `tsc --noEmit` under the hood